chromium_src/media/base/media_switches.cc

```diff
@@ -11,6 +11,7 @@ namespace media {
 
 OVERRIDE_FEATURE_DEFAULT_STATES({{
     {kLiveCaption, base::FEATURE_DISABLED_BY_DEFAULT},
+    {kEnableTabMuting, base::FEATURE_ENABLED_BY_DEFAULT},
```

Comment:
We probably want to add the feature to `BraveMainDelegateBrowserTest.EnabledFeatures` in `brave/app/brave_main_delegate_browsertest.cc`

Reply:
Good point, updated that browser test.